### PR TITLE
Dates

### PR DESCRIPTION
**Pull request by aghaynes**
Add date variables

### CODE REVIEW

**Comment by aghaynes** [comment]
tests added and a minor change to ensure that empty variables are also converted to dates

File: R/dates_secuTrial.R [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  datevars <- datevars[datevars %in% names(data)]
+  # print(datevars)
+  if(length(datevars) > 0) {
+    for(x in datevars) {
+      newdatecol <- dates_secuTrial(data[, x], format)
+      # print(head(newdatecol))
+      data[, paste0(x, ".date")] <- newdatecol
+    }
+  } else {
+    warning(paste("no dates detected in", get("obj", envir = parent.frame())))
```
Comment (PatrickRWright) [inline R/dates_secuTrial.R]: Sure you want to throw a warning here? Warnings usually throw me off a little. I checked it and saw that the warning is nothing to be concerned about.

**Comment by PatrickRWright** [comment]
Date times are not yet being parsed -> ctu05baseline$hiv_date has one example.

File: R/dates_secuTrial.R (hunk 1 above)
Comment (aghaynes) [reply R/dates_secuTrial.R]: it's useful for debugging as much as anything else... I'm not really bothered either way though

**Comment by aghaynes** [comment]
I added a note to the readme that datetimes are not yet handled. It would be the next logical step I suppose, but it's a bit more fiddly I think.

**Pull request merged**